### PR TITLE
Professional Learning Conditional options

### PR DESCRIPTION
**Pull request by juanmanzojr**
This PR adds conditions to the Professional Learning section of the expanded card. It will only display the professional learning section and its links if there are links to a professional learning program or self paced professional learning program. It also hides the professional learning section if the user is located outside the US.

### If the course offering has no Professional Learning program
Screenshot 2023-08-21 at 10 53 25 AM

### If the course offering only has one Professional Learning
Screenshot 2023-08-21 at 10 49 23 AM

Screenshot 2023-08-21 at 10 50 01 AM

### If the course offering has both
Screenshot 2023-08-21 at 10 50 36 AM

### If user is in US and in non-english

Screenshot 2023-08-22 at 12 23 37 PM

## Links

- jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?modal=detail&selectedIssue=ACQ-661)

## Testing story
Local

## Follow-up work
I will work on getting the corresponding links for the Available Resources section.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by TurnerRiley — APPROVED** [review]
Nice work! @dmcavoy can you confirm if using `isEnglish` is sufficient for the spec? I told Juan he could use this as I remember a while back someone telling me there's very few places we actually check where a user is by location or IP so we use language instead as a guess-timation?

**Comment by dmcavoy** [comment]
How much more effort would it be to try to use the location information (although not super accurate) that we have? Ideally we would still show the PL options to non english users in the US

**Comment by TurnerRiley** [comment]
> How much more effort would it be to try to use the location information (although not super accurate) that we have? Ideally we would still show the PL options to non english users in the US

We could use the locale code and see if it's specifically `en_us` so that we know they're English and in the U.S. (since don't we only have PL in English?)?

**Comment by dmcavoy** [comment]
I thought we had another way to look at someones location. I think Meg had conversations with Platform about this before.

**Comment by dmcavoy** [comment]
I don't think we have to block this PR on it but it would be good to investigate

**Comment by davidsbailey** [comment]
here is a PR of Meg's where we determine whether the user is in the US or not: https://github.com/code-dot-org/code-dot-org/pull/51481

File: apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx [hunk 1]
```diff
@@ -45,6 +46,7 @@ const ExpandedCurriculumCatalogCard = ({
   };
 
   const devices = JSON.parse(deviceCompatibility);
+  console.log(isInUS);
```
Comment (TurnerRiley) [inline apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx]: Please remove the log :)

File: apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx [hunk 2]
```diff
@@ -185,6 +187,7 @@ CurriculumCatalog.propTypes = {
   curriculaData: PropTypes.arrayOf(curriculumDataShape),
   isEnglish: PropTypes.bool.isRequired,
   languageNativeName: PropTypes.string.isRequired,
+  isInUS: PropTypes.bool,
```
Comment (TurnerRiley) [inline apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx]: Nit: probably want to make this required too!

**Review by TurnerRiley — APPROVED** [review]
Just a couple last nits but this looks great!

**Pull request merged**